src/iframe-worker.js

```diff
@@ -66,7 +68,7 @@ class IframeWorker extends EventTarget {
   }
 
   terminate() {
-    window.removeListener('message', this.handleIframeMessage)
+    window.removeEventListener('message', this.handleIframeMessage)
     document.removeChild(this.iframe)
```

Comment:
I was also having issues with this line; it usually complains that `this.iframe` isn't in the document tree :/

Reply:
I'll add a check to see if it is the tree before trying to remove it

Reply:
> I'll add a check to see if it is the tree before trying to remove it

Oh yes sorry I missed that!